src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -2074,6 +2075,7 @@ shared_ptr<runtime::Executable>
         case OP_TYPEID::HardSigmoid:
         case OP_TYPEID::LeakyRelu:
         case OP_TYPEID::LSTMCell:
+        case OP_TYPEID::MatMul:
```

Comment:
Please add "case OP_TYPEID::MatMul:" into switch of is_supported_impl function (line 2187 of this file) to be able to decompose MatMul Op for INTELGPU backend.

Reply:
Done